CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+* Prevent PHP fatal error for malformed date strings in PO files.
+
+### Changed
+* Only install updates for actually available languages.
+* Make closures static where possible.
+* Update PHPUnit to version 7.
+
 ## [2.0.0] - 2019-03-15
+
+### Fixed
+* Fix and improve various documentation.
```

Comment:
```suggestion
* Various documentation improvements.
```

Reply:
Or `Fix and improve various docs` perhaps. `Various documentation` sounds wrong though.